src/python/pants/goal/run_tracker.py

```diff
@@ -387,15 +390,51 @@ def store_stats(self):
     if target_data:
       run_information['target_data'] = ast.literal_eval(target_data)
 
+    def get_option(scope_or_scope_with_flag, flag=None):
+      """Looks up a scope in the options parsed by Pants.
+
+      scope_or_scope_with_flag is a string either either of form scope or scope.flag
+      If it's a scope, the scope's values will be gotten as a dict, and either returned if flag is
+        None, or else the flag will be looked up inside the dict.
+      If it's a scope.flag, the scope will be looked up and the flag looked up and returned.
+      If the scope or flag can't be found, ValueError will be raised.
+      """
+      scope_to_look_up = scope_or_scope_with_flag if scope_or_scope_with_flag != 'GLOBAL' else ''
+      try:
+        # Yeah, it's a little sketchy to be looking up Subsystem._options here, but... Someone went
```

Comment:
Instead of this, maybe `LocalPantsRunner` can pass its `self._options` (which is the whole-run options instance) into `Reporting.initialize()`, which can pass it through to `RunTracker.initialize()`.

Reply:
Done